core/domain/customization_args_util_test.py

```diff
@@ -390,9 +390,9 @@ def test_frontend_customization_args_defs_coverage(self):
 
             # Search for XCustomizationArgs interfaces and extract X,
             # where X is a interaction id.
-            # Group 1: The characters 'interface'.
+            # Group 1: The match for 'interface'.
             # Group 2: The interaction id.
-            # Group 3: The characters 'CustomizationArgs'.
+            # Group 3: The match for 'CustomizationArgs'.
             # Group 4: A space or an open bracket.
```

Comment:
Looks better, just one last suggestion to keep things a bit more consistent (please apply to other comments as well):

```suggestion
            # Group 1: Matches the string 'interface '.
            # Group 2: Matches an interaction id.
            # Group 3: Matches the string 'CustomizationArgs'.
            # Group 4: Matches a space or an open bracket.
```

Reply:
Done